crates/extensions/tedge_log_manager/src/lib.rs

```diff
@@ -92,11 +92,14 @@ impl LogManagerBuilder {
 
         // creating tedge-log-plugin.toml
         let example_config = r#"# Add the list of log files that should be managed
-files = [
-#    { type = "mosquitto", path = '/var/log/mosquitto/mosquitto.log' },
-#    { type = "software-management", path = '/var/log/tedge/agent/software-*' },
-#    { type = "c8y_CustomOperation", path = '/var/log/tedge/agent/c8y_CustomOperation/*' }
-]"#;
+[[files]]
+type = "software-management"
+path = "/var/log/tedge/agent/software-*"
```

Comment:
Shouldn't we use the `config_dir` path (e.g. the variable) rather than a hard coded value?